pkg/apps/strategy/support/acceptor.go

```diff
@@ -1,84 +0,0 @@
-package support
-
-import (
-	"fmt"
-	"io"
-	"time"
-
-	corev1 "k8s.io/api/core/v1"
-	kerrors "k8s.io/apimachinery/pkg/api/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/util/wait"
-	"k8s.io/apimachinery/pkg/watch"
-	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
-)
-
-// NewAcceptAvailablePods makes a new acceptAvailablePods from a real client.
-func NewAcceptAvailablePods(
```

Comment:
why did this go away?

Reply:
code is in pkg/apps/strategy/support/lifecycle.go